inference/models/paligemma/paligemma.py

```diff
@@ -150,6 +161,36 @@ def download_model_artefacts_from_s3(self) -> None:
         raise NotImplementedError()
 
 
+class LoRAPaliGemma(PaliGemma):
```

Comment:
Could you post a docs example + description on LoRA model?
I guess this class is needed to be able to load LoRA-fine-tuned models from hf hub as people are posting those, but what about our platform? PaliGemma is `RoboflowInferenceModel`, but it seems that we don't load weights from our hosting - which may be the indication that this is kind of "core" model? 
Also - is that always required to have HF token - or maybe we could rely on their auth?

Reply:
I can write up something more detailed but the short answer is this

1. LoRA is a technique to train a small "diff" from some base model A
2. This PR assumes that users will deploy the LoRA (the diff) to Roboflow, but in order to use it, they will need to download the base model A from huggingface
3. This doesn't reduce the amount of data transferred on the first LoRA loaded, but will significantly reduce data transfer on subsequent LoRA loads -- from 6GB for a fully finetuned model, to only 28MB for a new LoRA
4. This will also reduce our storage needs because we don't need to host 6GB for each fine tune, just 28MB for each LoRA

Reply:
So to be clear, we are loading weights from our hosting

Reply:
> Also - is that always required to have HF token - or maybe we could rely on their auth?

I'm not sure I understand this question

Reply:
from previous answers I see that HF tokens will be required, at least sometimes